### PR TITLE
FRONT-1537: Live data driven "Network overview" page

### PR DESCRIPTION
**Pull request by mondoreale**
I may have reinvented some wheels here. Hopefully not many and if some, then I'm quite confident they're solid wheels. Long story short, the overview page is as alive as can be with the current set of provided data.

- [x] Make sure long values don't ruin `StatGrid` layout.

### CODE REVIEW

**Comment by mondoreale** [comment]
Have a look guys. I'll merge it soon and eliminate any dups I made here progressively as I go. More coming!

File: src/hooks/operators.ts [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+        return () => {
+            mounted = false
+        }
+    }, [addr])
+
+    return stats
+}
+
+export function useDelegacyForWalletQuery(
```
Comment (pmichalowski) [inline src/hooks/operators.ts:138]: Delegacy? Maybe let's use Delegations to keep internal language consistency?

**Review by pmichalowski — APPROVED** [review]
Nice cleanup!

File: src/hooks/operators.ts (hunk 1 above)
Comment (mondoreale) [reply src/hooks/operators.ts:138]: Hm, maybe maybe.. It is true that it is kind of a new word. Delegacy = body of delegation collective. If it got you cornered a bit into thinking wtf than maybe it is me trying too hard after all. I'll go back to plurals.

File: src/types/index.ts [hunk 2]
```diff
@@ -1,7 +1,43 @@
...
+/**
+ * `ParsedOperator` enhanced with `apy` and `myShare`. It is *not* Delegation
+ * structure coming from the Graph directly in any way.
+ */
+export interface Delegation extends ParsedOperator {
+    apy: number
+    myShare: BN
+}
+
+export interface DelegacyStats {
```
[the remark this answers is not in the document]
Comment (mondoreale) [reply src/types/index.ts:20]: See above.

**Pull request merged**